### PR TITLE
Community Aside Block

### PR DESCRIPTION
**Pull request by PolinaShneider**
* Add db record
* Add field to Model
* Process in Controller
* Add custom template
* Add CSS

### CODE REVIEW

File: www/application/views/main.php [hunk 1]
```diff
@@ -30,16 +30,16 @@
...
-        $bodyModifiers[] = 'body--with-branding';
-    }
-
-    if (!empty($contentOnly)) {
-        $bodyModifiers[] = 'body--content-only';
-    }
+<?
+    $bodyModifiers = [];
+
+    if (!empty($site_info['branding']) && empty($hideBranding)) {
```
Comment (khaydarov) [inline www/application/views/main.php]: `php

$bodyModifiers = [
   [!empty($site_info['branding']) && empty($hideBranding), 'body--with-branding'],
  [!empty($contentOnly), 'body--content-only'],
[ ajsdkjansm, 'asdasd']
]

foreach($bodyModifiers as $modifiers) {
  if ($modifiers[0]) {
    $actualModifiers[] = $modifiers[1];
  }
}`

File: www/application/views/main.php [hunk 2]
```diff
@@ -30,16 +30,16 @@
...
-
-    if (!empty($site_info['branding']) && empty($hideBranding)) {
-        $bodyModifiers[] = 'body--with-branding';
-    }
-
-    if (!empty($contentOnly)) {
-        $bodyModifiers[] = 'body--content-only';
-    }
+<?
+    $bodyModifiers = [];
```
Comment (khaydarov) [inline www/application/views/main.php]:
```php
$bodyModifiers = [
   [!empty($site_info['branding']) && empty($hideBranding), 'body--with-branding'],
   [!empty($contentOnly), 'body--content-only'],
   [$condition, '$css'],
   ...
]

foreach($bodyModifiers as $modifiers) {
  if ($modifiers[0]) {
    $actualModifiers[] = $modifiers[1];
  }
}`

File: www/public/app/css/components/community-aside.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.community-aside {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 30px;
```
Comment (neSpecc) [inline www/public/app/css/components/community-aside.css]: --islandPadding

File: www/public/app/css/components/community-aside.css [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+         width: 90px;
+         height: 90px;
+         border-radius: 3px;
+         border: 1px solid var(--lightGrayLine);
+         margin-bottom: 20px;
+
+        &--default {
+             width: 45px;
+             height: 45px;
+             padding: 21.5px;
```
Comment (neSpecc) [inline www/public/app/css/components/community-aside.css]: у img нет паддингов

**Pull request merged**